.github/workflows/main.yml

```diff
@@ -12,12 +12,18 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        flutter: [ '3.24.0', '3.27.0', '3.29.0', '3.32.0' ]
+        flutter: [ '3.24.0', '3.27.0', '3.29.0', '3.32.0', 'stable', 'beta' ]
```

Comment:
Currently if we run this on "stable", we run it twice? (stable is 3.32.x at the moment)